layouts/partials/header-menu.html

```diff
@@ -1,4 +1,27 @@
 <ul>
-    <li><a href="{{ "about" | relURL }}">{{ i18n "about" }}</a></li>
-    <li><a href="{{ "/" | relURL }}">{{ i18n "posts" }}</a></li>
```

Comment:
These i18n entries can now be removed from all the i18n files as well.

Reply:
Removed.